.genx/package.json

```diff
@@ -35,6 +35,7 @@
           ".sh",
           ".json",
```

Comment:
For the PR title you want to follow conventional commits format: https://www.conventionalcommits.org/en/v1.0.0/#summary

E.g. `feat: sample event handler test JIRA-123`